ext/dispatcher.go

```diff
@@ -127,7 +128,7 @@ func NewDispatcher(opts *DispatcherOpts) *Dispatcher {
 	var errHandler DispatcherErrorHandler
 	var panicHandler DispatcherPanicHandler
 	var unhandledErrFunc ErrorFunc
-	var errLog *log.Logger
+	logger := slog.New(slog.NewTextHandler(os.Stderr, nil))
```

Comment:
I think it could be beneficial to use the `slog.Default()` here, so that if the caller had initialised the slog.Default logger before calling NewDispatcher, would be automatically propagated.

Reply:
Great catch, thank you! Had forgotten about that entirely :)